From 7d2d6c41f6ccb625e61414469557900066c90845 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Thu, 26 Jul 2018 10:56:12 +0800
Subject: [PATCH 1345/1345] arm64: dts: marvell: enable pmu dts node in ap806
 config space

Enable pmu unit to support perf feature on Marvell Armada platform.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index 9edb4e0..0f96255 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -222,7 +222,6 @@
 				compatible = "arm,cortex-a72-pmu";
 				interrupt-parent = <&pic>;
 				interrupts = <17>;
-				status = "disabled";
 			};
 
 			ccu-spmu {
-- 
1.7.9.5

